sponsors/index.html

```diff
@@ -13,111 +12,231 @@
-        });
+      $(function () {
+        $("#navbar").load("/components/navbar.html");
+      });
     </script>
 
     <section id="sponsors">
-        <h1 align="center" class="content-header">Sponsors</h1>
-
-        <div class="sponsor-level">
-            <div class="sponsor-detail">
-                <a href="https://stripe.com/jobs/university" target="_blank">
-                    <div class="sponsor-image">
-                        <img src="/assets/img/logos/sponsors/stripe.png" alt="Stripe logo">
-                    </div>
-                </a>
-
-                <div class="level-indicator platinum">
-                    PLATINUM
-                </div>
-            </div>
-            <div class="sponsor-detail">
-                <a href="https://www.uber.com/us/en/careers/" target="_blank">
-                    <div class="sponsor-image">
-                        <img src="/assets/img/logos/sponsors/uber.png" alt="Uber logo">
-                    </div>
-                </a>
-
-                <div class="level-indicator platinum">
-                    PLATINUM
-                </div>
+      <h1 align="center" class="content-header">Sponsor Us!</h1>
+
+      <div align="center" class="sponsor-call">
+        We are committed to cultivating diverse voices in our student body while
+        maintaining engagement between company representatives and our members
+        in whatever setting that may be. With your contribution, we will plan
+        in-person and hybrid opportunities through which we aim to help members
+        gain confidence in pursuing their ambitions. In turn, you will be able
+        to connect with these dedicated students and gain more recognition
+        within our organization!
+      </div>
+
+      <h1 align="center" class="content-header" style="padding-bottom: 20px">
+        Sponsorhip Tiers:
+      </h1>
+
+      <div align="center" style="" class="sponsor-benefits color-indicator">
+        <span style="display: inline" class="color-indicator platinum"
+          >PLATINUM</span
+        >
+        <span style="display: inline" class="color-indicator gold">GOLD</span>
+        <span style="display: inline" class="color-indicator silver"
+          >SILVER
+        </span>
+        <span style="display: inline" class="color-indicator bronze"
+          >BRONZE</span
+        >
+      </div>
+
+      <div
+        class="sponsor-call"
+        style="vertical-align: middle; padding-top: 13px"
+      >
+        <li class="dropdown" style="text-align: center">
+          <div class="dropdown" style="text-align: center">
+            <a
+              href="#"
+              class="dropdown-toggle"
+              data-toggle="dropdown"
+              role="button"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
+              <span class="dropdown-lg-view"
+                ><u
+                  >__________________Click here to learn
```

Comment:
is there a way to remove these lines? maybe consider changing the text into a button & setting the button width to be a certain value to avoid shrinking the dropdown image